Source/ZoomNet/Resources/Webinars.cs

```diff
@@ -120,6 +120,53 @@ public Task<ScheduledWebinar> CreateScheduledWebinarAsync(string userId, string
 				.AsObject<ScheduledWebinar>();
 		}
 
+		/// <summary>
+		/// Updates an existing scheduled webinar.
+		/// </summary>
+		/// <param name="webinarId">The webinar ID.</param>
+		/// <param name="topic">Webinar topic.</param>
+		/// <param name="agenda">Webinar description.</param>
+		/// <param name="start">Webinar start time.</param>
+		/// <param name="duration">Webinar duration (minutes).</param>
+		/// <param name="password">Password to join the webinar. By default the password may only contain the following characters: [a-z A-Z 0-9 @ - _ *]. Max of 10 characters. This can be updated within Zoom account settings.</param>
+		/// <param name="settings">Webinar settings.</param>
+		/// <param name="trackingFields">Tracking fields.</param>
+		/// <param name="cancellationToken">The cancellation token.</param>
+		/// <returns>
+		/// The async task.
+		/// Zoom will return an HTTP 200 response when there is no webinar subscription present.
+		/// A successful update is reported by an HTTP 204 value.
+		/// </returns>
+		public Task UpdateScheduledWebinarAsync(long webinarId, string topic, string agenda, DateTime start, int duration, string password = null, WebinarSettings settings = null, IDictionary<string, string> trackingFields = null, CancellationToken cancellationToken = default)
```

Comment:
I missed this during my review yesterday: all these parameters must be optional so developers can selectively update values without necessarily having to provide all the other ones. For instance, we need string topic = null, DateTime? start = null and so forth in order to allow updating a value like the agenda for example without having to provide the topic, the start time, etc.

These values are necessary when you create a webinar but they are optional when updating an existing webinar.

Reply:
Done